### ISSUE TITLE
Use with neovim + python3 results in some kind of new error I haven't seen before

### ISSUE THREAD

**Issue by arooni** [tools-life/taskwiki#267]
ENV:  Ubuntu 18.04.  

happens on nvim startup.  ensured i have the latest version of all plugins via :PlugUpdate first.

```
[I]  ~  nvim --version
NVIM v0.3.8
Build type: Release
LuaJIT 2.1.0-beta3
Compilation: /usr/bin/cc -g -O2 -fdebug-prefix-map=/build/neovim-lpKDO8/neovim-0.3.8=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=1 -DDISABLE_LOG -Wdate-time -D_FORTIFY_SOURCE=1 -Wconversion -O2 -DNDEBUG -DMIN_LOG_LEVEL=3 -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fdiagnostics-color=auto -Wno-array-bounds -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -I/build/neovim-lpKDO8/neovim-0.3.8/build/config -I/build/neovim-lpKDO8/neovim-0.3.8/src -I/usr/include -I/build/neovim-lpKDO8/neovim-0.3.8/build/src/nvim/auto -I/build/neovim-lpKDO8/neovim-0.3.8/build/include
Compiled by team+vim@tracker.debian.org

Features: +acl +iconv -jemalloc +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info

#########################

Error detected while processing function provider#python3#Call:
line   18:
Traceback (most recent call last):
  File "/home/david/.vim/plugged/taskwiki/taskwiki/main.py", line 640, in <module>
    WholeBuffer.update_from_tw()
  File "/home/david/.vim/plugged/taskwiki/taskwiki/errors.py", line 26, in wrapped_function
    original_function(*args, **kwargs)
  File "/home/david/.vim/plugged/taskwiki/taskwiki/decorators.py", line 10, in wrapped_function
    original_function(*args, **kwargs)
  File "/home/david/.vim/plugged/taskwiki/taskwiki/main.py", line 44, in update_from_tw
    c.buffer.push()
  File "/home/david/.vim/plugged/taskwiki/taskwiki/cache.py", line 28, in push
    util.get_buffer(self.buffer_number)[:] = self.data
  File "/home/david/.local/lib/python3.6/site-packages/pynvim/api/buffer.py", line 65, in __setitem__
    return self.request('nvim_buf_set_lines', start, end, False, lines)
  File "/home/david/.local/lib/python3.6/site-packages/pynvim/api/common.py", line 58, in request
    return self._session.request(name, self, *args, **kwargs)
  File "/home/david/.local/lib/python3.6/site-packages/pynvim/api/nvim.py", line 182, in request
    res = self._session.request(name, *args, **kwargs)
  File "/home/david/.local/lib/python3.6/site-packages/pynvim/msgpack_rpc/session.py", line 102, in request
    raise self.error_wrapper(err)
pynvim.api.common.NvimError: String cannot contain newlines
Press ENTER or type command to continue
```

**Comment by nkakouros** [comment]
@arooni @tbabej This can be closed as a duplicate of #277.

While this issue is newer, #277 contains more information that this one here.

**Comment by tbabej** [comment]
Closing as duplicate.
